src/state/nftMarket/reducer.ts

```diff
@@ -148,112 +121,62 @@ export const fetchNftsFromCollections = createAsyncThunk(
 )
 
 /**
- * Fetch fresh marketdata for existing tokens in the store
+ * This action keeps data on the individual PancakeBunny page up-to-date. Operation is a twofold
+ * 1. Update existing NFTs in the state in case some were sold or got price modified
+ * 2. Fetch 30 more NFTs with specified bunny id
  */
-export const updateNftTokensData = createAsyncThunk<
+export const fetchNewPBAndUpdateExisting = createAsyncThunk<
```

Comment:
Getting new NFTs and updating old ones are combined into 1 hook to not have a racing game between two update mechanisms.